tox.ini

```diff
@@ -1,111 +1,32 @@
 [tox]
-envlist=py27-dj14,py27-dj15,py26-dj14,py26-dj15,py27-dj16
+envlist=
+    # core python/django/database
+    py26-dj{14,15}-{sqlite,mysql,postgres},
+    py27-dj{14,15,16}-{sqlite,mysql,postgres},
+    # contrib: widgy_mezzanine
+    py{26,27}-dj{14,15,16}-mezzanine,
+    # javascript
+    javascript
 
 [testenv]
-commands=/usr/bin/env make test
-
-[testenv:py27-dj14]
-basepython=python2.7
-deps =
-  Django==1.4
-
-[testenv:py27-dj15]
-basepython=python2.7
-deps =
-  https://www.djangoproject.com/download/1.5c1/tarball/
-
-[testenv:py26-dj14]
-basepython=python2.6
-deps =
-  Django==1.4
-
-[testenv:py26-dj15]
-basepython=python2.6
-deps =
-  https://www.djangoproject.com/download/1.5c1/tarball/
-
-[testenv:py27-dj16]
-basepython=python2.7
-deps =
-  --editable=git+https://github.com/django/django@master#egg=django
-
-[testenv:py26-dj15-mysql]
-basepython=python2.6
-deps =
-  https://www.djangoproject.com/download/1.5c1/tarball/
-  MySQL-python==1.2.3
-setenv =
-  DATABASE_ENGINE=mysql
-
-[testenv:py27-dj15-mysql]
-basepython=python2.7
-deps =
-  https://www.djangoproject.com/download/1.5c1/tarball/
-  MySQL-python==1.2.3
-setenv =
-  DATABASE_ENGINE=mysql
-
-[testenv:py27-dj14-mysql]
-basepython=python2.7
-deps =
-  Django==1.4
-  MySQL-python==1.2.3
-setenv =
-  DATABASE_ENGINE=mysql
-
-[testenv:py27-dj16-mysql]
-basepython=python2.7
-deps =
-  --editable=git+https://github.com/django/django@master#egg=django
-  MySQL-python==1.2.3
-setenv =
-  DATABASE_ENGINE=mysql
-
-[testenv:py26-dj14-mysql]
-basepython=python2.6
-deps =
-  Django==1.4
-  MySQL-python==1.2.3
-setenv =
-  DATABASE_ENGINE=mysql
-
-
-[testenv:py26-dj15-postgres]
-basepython=python2.6
-deps =
-  https://www.djangoproject.com/download/1.5c1/tarball/
-  psycopg2
-setenv =
-  DATABASE_ENGINE=postgres
-
-[testenv:py27-dj15-postgres]
-basepython=python2.7
-deps =
-  https://www.djangoproject.com/download/1.5c1/tarball/
-  psycopg2
-setenv =
-  DATABASE_ENGINE=postgres
-
-[testenv:py27-dj14-postgres]
-basepython=python2.7
-deps =
-  Django==1.4
-  psycopg2
-setenv =
-  DATABASE_ENGINE=postgres
-
-[testenv:py26-dj14-postgres]
-basepython=python2.6
-deps =
-  Django==1.4
-  psycopg2
-setenv =
-  DATABASE_ENGINE=postgres
-
-[testenv:py27-dj16-postgres]
-basepython=python2.7
-deps =
-  --editable=git+https://github.com/django/django@master#egg=django
-  psycopg2
-setenv =
-  DATABASE_ENGINE=postgres
+basepython=
+  py26: python2.6
+  py27: python2.7
+commands=
+  mezzanine: pip install -e .[page_builder,widgy_mezzanine,form_builder] --log-file {envdir}/pip-extras-require-log.log
+  /usr/bin/env make test-py
+deps=
+  dj14,dj15,dj16: South==1.0.2
+  dj14: Django>=1.4,<1.5
+  dj15: Django>=1.5,<1.6
+  dj16: Django>=1.6,<1.7
+  sqlite: pytest
```

Comment:
oh, interesting. could you comment that? is there any project that provides something like cats.py? just a noop?